Sources/NestKit/GitHub/ExcludedVersion.swift

```diff
@@ -0,0 +1,19 @@
+/// {owner}/{repo} and (optional) version pairs to exclude
+public struct ExcludedVersion: Equatable, Sendable {
+    /// A reference to a repository.
+    /// {owner}/{repo}
+    public let reference: String
+    public let version: String?
+
+    public init?(argument: String) {
+        let split = argument.split(separator: "@")
+        guard split.count == 1 || split.count == 2 else { return nil }
+        self = .init(reference: String(split[0]), version: split.count == 2 ? String(split[1]) : nil)
+    }
```

Comment:
This is a concern of not `NestKit` but `nest`.
Please move this initializer to `net` module.

For this, it's OK to make `init(referent:version)` a public function.

Reply:
Thanks!
I've made the corrections here: https://github.com/mtj0928/nest/pull/33/commits/7fdac4082f2b8cd78e6a48d30ba836761e25b250